Integrations/Tanium/Tanium.py

```diff
@@ -302,6 +302,9 @@ def deploy_action(handler):
 
     for key, value in demisto.args().items():
         kwargs[key] = value
+
+    kwargs["get_results"] = True if kwargs.get('get_results').lower() == 'true' else False
```

Comment:
My bad for the first time, it might raise an exception if you don't enter get_results at all.
```suggestion
    kwargs["get_results"] = True if kwargs.get('get_results', '').lower() == 'true' else False
```